src/material/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -324,3 +372,27 @@ export class MatTabLink extends _MatTabLinkBase implements OnDestroy {
     this._tabLinkRipple._removeTriggerEvents();
   }
 }
+
+/**
+ * Tab panel component associated with MatTabNav.
+ */
+@Component({
+  selector: 'mat-tab-nav-panel',
+  exportAs: 'matTabNavPanel',
+  template: '<ng-content select="router-outlet"></ng-content>',
```

Comment:
I'm not sure that we should be limiting this to `router-outlet`. People could be using a different router implementation.

Reply:
Done.